src/org/jitsi/impl/neomedia/rtcp/RTCPTCCPacket.java

```diff
@@ -0,0 +1,322 @@
+    {
+        return getPacketsFci(getFCI(baf));
+    }
+
+    /**
+     *
+     * @param next
+     * @return
+     */
+    public static TreeMap<Integer, Long> getPacketsFci(ByteArrayBuffer fciBuffer)
+    {
+        TreeMap<Integer,Long> packets
+            = new TreeMap<>(RTPUtils.sequenceNumberComparator);
+        if (fciBuffer == null)
+        {
+            return packets;
+        }
+
+        byte[] buf = fciBuffer.getBuffer();
+        int off = fciBuffer.getOffset();
+        int len = fciBuffer.getLength();
+
+        int baseSeq = RTPUtils.readUint16AsInt(buf, off);
+        int packetStatusCount = RTPUtils.readUint16AsInt(buf, off + 2);
+
+        // reference time with resolution 250µs
+        long referenceTime = RTPUtils.readUint24AsInt(buf, off + 4) << 8;
+
+        off += 8;
+
+        int deltaOff = 3;
+
+        return packets;
+    }
+
+    /**
+     * The value of the "fmt" field for a transport-cc packet.
+     */
+    public static final int FMT = 15;
+
+    /**
+     * The map which contains the sequence numbers (mapped to the reception
+     * timestamp) of the packets described by this RTCP packet.
+     */
+    private TreeMap<Integer, Long> packets = null;
+
+    /**
+     * Initializes a new <tt>NACKPacket</tt> instance.
+     * @param base
+     */
+    public RTCPTCCPacket(RTCPCompoundPacket base)
+    {
+        super(base);
+    }
+
+    /**
+     * Initializes a new {@link RTCPTCCPacket} instance with a specific "packet
+     * sender SSRC" and "media source SSRC" values, and which describes a
+     * specific set of sequence numbers.
+     * @param senderSSRC the value to use for the "packet sender SSRC" field.
+     * @param sourceSSRC the value to use for the "media source SSRC" field.
+     * @param packets the set of RTP sequence numbers and their reception
+     * timestamps which this packet is to describe.
+     * @param fbPacketCount the index of this feedback packet, to be used in the
+     * "fb pkt count" field.
+     *
+     * Note that this implementation is not optimized and might not always use
+     * the minimal possible number of bytes to describe a given set of packets.
+     * Specifically, it does take into account that sequence numbers wrap
+     * at 2^16 and fails to pack numbers close to 2^16 with those close to 0.
+     */
+    public RTCPTCCPacket(long senderSSRC, long sourceSSRC,
+                         TreeMap<Integer, Long> packets,
+                         byte fbPacketCount)
+    {
+        super(FMT, RTPFB, senderSSRC, sourceSSRC);
+
+        TreeSet<Map.Entry<Integer, Long>> sequenceNumbers
+            = (TreeSet) packets.entrySet();
+
+        Map.Entry<Integer, Long> first = sequenceNumbers.first();
+        int firstSeq = first.getKey();
+        Map.Entry<Integer, Long> last = sequenceNumbers.last();
+        int packetCount
+            = 1 + RTPUtils.sequenceNumberDiff(last.getKey(), firstSeq);
+
+        byte[] buf = new byte[(packetCount / 7 + 1) * 2 + 8];
```

Comment:
Seven packets per packet status chunk which is two bytes long plus an additional 8 bytes for the fixed fields, obviously. :)
I think there's a bug in here, will fix AND document.